components/gcp/container/component_sdk/python/kfp_component/google/bigquery/_query.py

```diff
@@ -24,6 +24,39 @@
 # TODO(hongyes): make this path configurable as a environment variable
 KFP_OUTPUT_PATH = '/tmp/kfp/output/'
 
+
+def query_only(query, project_id, dataset_location='US', job_config=None):
```

Comment:
Can the `query` function be used for this purpose? For example, it could work the same when `dataset_id` is `None`.